grails-app/views/project/_projectAdmin.gsp

```diff
@@ -136,6 +145,24 @@
                     <h3>Project Access</h3>
                     <g:render template="/admin/permissionTable" model="[loadPermissionsUrl:g.createLink(controller:'project', action:'getMembersForProjectId', id:project.projectId), removeUserUrl:g.createLink(controller:'user', action:'removeUserWithRoleFromProject'), entityId:project.projectId, user:user]"/>
                 </div>
+                <div id="edit-documents" class="pill-pane tab-pane">
```

Comment:
There is a big enough copy/paste here I think it's worth pulling this documents tab into a separate GSP fragment.